cli/docs/flags.go

```diff
@@ -56,6 +56,9 @@ const (
 	Sast      = "sast"
 	Secrets   = "secrets"
 	WithoutCA = "without-contextual-analysis"
+
+	// Sast related flags
+	SastRules = "sast-rules"
```

Comment:
```suggestion
	SastRules = "sast-rules-file"
```
consider changing the flag name (unless approved already)